### ISSUE TITLE
Add an invalid websites file

### ISSUE THREAD

**Issue by mehtaarn000** [sherlock-project/sherlock#870]
## Checklist

- [x] I'm reporting a feature request
- [x] I've checked for similar feature requests including closed ones

## Description

Countless times, I've looked through this project to find a popular website that hasn't been added yet (for example discord)
and after not finding it, I try to implement a method to add it. However, usually this website is invalid (ex. when you send an http request, it redirects it). Why not add a file that has a list of invalid websites?

**Comment by sdushantha** [comment]
Thank you for the suggestion but I don't feel like this is needed in this project. First of all, that would unnecessarily increase the size of this repo because we would be pretty much creating a database of invalid sites. Secondly, we do have the [removed_sites.md](https://github.com/sherlock-project/sherlock/blob/master/removed_sites.md) file which stores all the sites which were previously added to Sherlock but then had to be removed due to false results.

Just think of the youtube-dl project, it is able to download videos from hundreds of sites but at the same time, it is unable to download content from many other websites. They don't have a database of sites which they failed to find a downloading method for because it would unnecessarily increase the size of this repo. 

I do understand that if we did have database of sites which we failed to find a detecting method for, it would save more time for contributors because they would not be wasting time. But on the other hand, having such a file could lead to people being discouraged to finding a method for a website because they already have the thought that somone already tried and failed. This is could lead to them probably missing a method that actually works.
